extensions/pom.xml

```diff
@@ -2742,6 +2742,36 @@
       <artifactId>spring-security-crypto</artifactId>
       <version>5.8.12</version>
     </dependency>
+    <dependency>
+      <groupId>org.springframework.security</groupId>
+      <artifactId>spring-security-oauth2-client</artifactId>
+      <version>5.8.14</version>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.security</groupId>
+      <artifactId>spring-security-oauth2-core</artifactId>
+      <version>5.8.14</version>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.security</groupId>
+      <artifactId>spring-security-oauth2-jose</artifactId>
+      <version>5.8.14</version>
```

Comment:
Do not hard-code versions here. Use the spring security version property from the parent pom.